src/main/java/com/wasim/codefellowship/controllers/ApplicationUserController.java

```diff
@@ -40,14 +48,15 @@ public String getSignupPage() {
     public String etPhoneHome() {
         return "index.html";
     }
+
     @PostMapping("/signup")
     public RedirectView postSignup(String username, String password, String firstName, String lastName, String bio, LocalDate dateOfBirth) {
         ApplicationUser user = new ApplicationUser();
         user.setUsername(username);
         user.setFirstName(firstName);
         user.setLastName(lastName);
         user.setBio(bio);
-
+        user.setDateOfBirth(dateOfBirth);
```

Comment:
The signup form is passing in an integer for dateOfBirth, but your postSignup method is expecting a LocalDate object. Using the "date" input type in your form is probably the most efficient way to address that; you could also do some processing in the postSignup method to convert the integer into a date.